### PR TITLE
Add name replacement for anons

### PR DESCRIPTION
**Pull request by Wolvan**
Because I had multiple asks about this and a few of my friends asked as well

### CODE REVIEW

**Comment by bvtsang** [comment]
This needs to handle the anonymous icons from asks in the inbox.

File: Extensions/satsukimous.js [hunk 1]
```diff
@@ -52,7 +67,7 @@ XKit.extensions.satsukimous = new Object({
 		}
 		$( "img" ).filter(function( index ) {
 			return $( this ).attr( "src" ).indexOf( "anonymous_avatar" ) !== -1;
-		}).attr( "src", replacement ).addClass("satsukimous_src matoiRYUKOOOOoO");
+		}).attr( "src", replacement ).addClass("satsukimous_src matoiRYUKOOOOoO").parent().parent().find(".asker > .name").text(XKit.extensions.satsukimous.preferences.replace_name.value ? XKit.extensions.satsukimous.preferences.name_replacement.value : "anonymous");
```
Comment (bvtsang) [inline Extensions/satsukimous.js]: Nitpick: place your addition on the next line, mostly because I don't want to scroll all the way to the right to debug

**Comment by blackjackkent** [comment]
RIght now this defaults to "satsuki" if the field is blank; I would suggest we default it to "Anonymous" instead.

Other than that, LGTM.

**Comment by blackjackkent** [comment]
LGTM

@homu r+

**Pull request merged**